draft-sharabayko-srt.md

```diff
@@ -1966,20 +1966,18 @@ have no chance to be delivered in time, and allows the receiver to skip missing
 that have not been delivered in time. The timeout of dropping a packet is based on the 
 TSBPD mechanism ({{tsbpd}}).
 
-With TLPKTDROP is enabled, when a packet timestamp is older than TLPKTDROP_THRESHOLD,
-the packet is considered "Too-Late" to be delivered. Thus, the sender might need to drop
-the packet.
+When TLPKTDROP mechanism is enabled, a packet is considered "too late" to be
+delivered and may be dropped by the sender if the packet timestamp is older than
+TLPKTDROP_THRESHOLD.
 
-TLPKTDROP_THRESHOLD: a marginal value that defines a limit for when a packet is considered
-to be delivered in time. 
-: The value is related to SRT Latency ({{srt-latency}}). For an efficient packet drop mechanism,
-  it is recommended that a higher value than the latency be used. This will allow the receiver
-  to drop the packet first so that the sender drops the packet only when no proper response is
-  received from the peer (due to severe congestion, for example). When considering RTT, it is
-  recommended that the threshold value be 1.25 times the SRT Latency value.
+TLPKTDROP_THRESHOLD is related to the SRT latency ({{srt-latency}}). For an effective functioning of
+the Too-Late Packet Drop mechanism, it is recommended that a higher value than the SRT latency is used.
+This will allow the SRT receiver to drop missing packets first while the sender drops packets if
+only no proper response is received from the peer in time (e.g., due to severe congestion).
+The recommended threshold value is 1.25 times the SRT latency value.
```

Comment:
```suggestion
TLPKTDROP_THRESHOLD is related to SRT latency ({{srt-latency}}). For the Too-Late Packet Drop mechanism to function effectively, it is recommended that a value higher than the SRT latency is used.
This will allow the SRT receiver to drop missing packets first while the sender drops packets if a proper response is not received from the peer in time (e.g., due to severe congestion).
The recommended threshold value is 1.25 times the SRT latency value.
```